### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by aemiers** [itsnameissteven/Refactor-Tractor-Fitlit#17]
- [x] You must be able to tab through your app and use it without a mouse
- [x] Your app must still be viewable when tested with a colorblind extension
- [x] You must score as close to 100% in an Accessibility Audit as possible. Be prepared to explain any accessibility audits your application is failing.
- [x] Your HTML must be written semantically and ARIA tags should be used (ONLY if needed / appropriate)
